spec/features/featured_item_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'spec_helper'
+
+RSpec.describe "featuring items" do
+  context "when viewing a featured item" do
+    let(:user) do
+      u = FactoryGirl.build(:user)
+      u.group_list = "admin"
+      u.save
+      u
+    end
+    it "should have a working unfeature link", :js => true do
```

Comment:
Yes, leave it in. You'll need it now that #1206 is in the queue. @luisgreg99 if you could remove the hashrocket, i.e. `js: true`, we're good to go. You know, because @mjgiarlo.  :wink: